### ISSUE TITLE
What is the difference between batch["labels"] and batch["pos_labels"]

### ISSUE THREAD

**Issue by bryanzhou008** [swarnaHub/ExplagraphGen#2]
This is a very interesting paper, and great source code. In src/train_graph_gen_max_margin.py I am confused by the following code. Here what is the difference between batch["labels"] and batch["pos_labels"]. i.e. how are they generated?

```
def _step(self, batch: dict) -> Tuple:
        pad_token_id = self.tokenizer.pad_token_id
        src_ids, src_mask = batch["input_ids"], batch["attention_mask"]
        if "pos_labels" in batch:
            pos_tgt_ids, all_neg_tgt_ids = batch["pos_labels"], batch["all_neg_labels"]
        else:
            pos_tgt_ids, all_neg_tgt_ids = None, None
        tgt_ids = batch["labels"]
```
